doc/schema.sql

```diff
@@ -1,5 +1,6 @@
 DROP TABLE IF EXISTS schema_version;
```

Comment:
Lets remove all these drop statements. We won't want to drop the tables going into qa and prod